common/app/model/content.scala

```diff
@@ -272,15 +272,8 @@ final case class Content(
       ("atoms", JsArray(atomIdentifiers))
     }
 
-    val atomTypesMeta = atoms.map { atoms =>
-      val atomTypes = Map(
-        "guide" -> JsBoolean(!atoms.guides.isEmpty),
-        "profile" -> JsBoolean(!atoms.profiles.isEmpty),
-        "qanda" -> JsBoolean(!atoms.qandas.isEmpty),
-        "timeline" -> JsBoolean(!atoms.timelines.isEmpty)
-      )
-      
-      ("atomTypes", JsObject(atomTypes))
+    val atomTypesMeta = atoms.map(_.atomTypes).map { atomTypes =>
```

Comment:
Is the double `map` necessary here?

Reply:
can you please comment [there](https://github.com/guardian/frontend/pull/18222) instead? fair question. I wonder if the Scala compiler handles deforestation.